website/docs/components/page-header/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,237 @@
+
+![Page Header Actions Tertiary Button](/assets/components/page-header/page-header-actions-tertiary-button.png =700x*)
+
+### Dropdowns
+
+Use a [Dropdown](/components/dropdown) in the Page Header to combine multiple secondary functions, tasks, or elements that assist the user in managing the information or objects on the page. Examples of this include:
+
+- Copying API credentials to manage the object remotely
+- Editing or changing settings
+- Deleting or deactivating an object
+- Linking to documentation
+
+When used in these scenarios and paired with a `primary` button, use a `secondary` [Dropdown](/components/dropdown).
+
+!!! Do
+
+Use a primary [Dropdown](/components/dropdown) when the same action can be done in multiple different contexts. For example, taking the same action through a different interface or path; GUI, CLI, or API call.
+
+![Page Header primary dropdown](/assets/components/page-header/page-header-actions-do-primary-dropdown.png =853x*)
+
+!!!
+
+!!! Dont
+
+Don't use a `primary` [Dropdown](/components/dropdown) when combining different, unrelated actions. Instead, use the `secondary` color.
+
+![Page Header primary dropdown](/assets/components/page-header/page-header-actions-dont-primary-dropdown.png =853x*)
+
+!!!
+
+### Action color and pairing
+
+While the Page Header supports up to three actions in Figma, we recommend limiting the number of actions to two for most pages. These actions should generally consist of a primary action (e.g. "Create" or "Deploy") and a secondary action (e.g., "Manage").
+
+For more details on how to combine, order, and organize buttons, refer to the [Button order, organization, and alignment](/patterns/button-organization) pattern documentation.
+
+!!! Dont
+
+Don't pair two primary actions in a Page Header. Instead, communicate the highest priority action for the user with a primary Button or Dropdown and other actions with a secondary Button or Dropdown.
+
+![Page Header two primary actions](/assets/components/page-header/page-header-dont-two-actions.png =700x*)
+
+!!!
+
+## Responsiveness
+
+The Page Header component in Figma supports two sizes; `large` which accounts for the majority of desktop sizes and large tablets, and `small` which accounts for smaller tablet and mobile devices. The core difference between each variant is the vertical stacking of elements.
+
+The Ember component uses a variety of different methods to ensure fluidity and responsiveness:
+
+- By default, the component will fill the page layout it is used within. It does not have any padding or margin explicitly applied, allowing it to adapt to different layout and spacing methods.
+- The component has breakpoints by means of container queries (at `7680px` and one at `400px`) that account for the majority of content within the component and stack elements in a single column as the container shrinks.
```

Comment:
Can we double-check the first breakpoint of 7680px?? I saw a different defition in the [Vercel deployment ](https://hds-showcase-g6krxiq38-hashicorp.vercel.app/components/page-header)that says:

> The component uses container queries to determine two breakpoints at 700px and 400px which stack the content in a single column.

Reply:
@apriofrost I've had the same doubt, and looking at the code seems the correct one is `768px` (so the `700px` is the one to change)

Reply:
Good catch!